cmake/libxmp-sources.cmake

```diff
@@ -0,0 +1,173 @@
+set(LIBXMP_SRC_LITE)
```

Comment:
My experience with CMake is essentially none (I had to edit libmodplug's CMakeLists.txt once, does that count)? Is this generally the standard way of grouping sources, or is there even a standard way? My only concern here is another project I work on hides its MSVC and Xcode projects/object lists in subdirs and often I forget to update them when adding/renaming files.

Reply:
There are MANY ways of the source code listing and they were used in various situations:
- Just list all sources at the `add_executable()` or `add_library()` call, the default way
- Build arrays of sources to make them being conveniently managed when they shared between multiple targets (the example - shared and static build of the same library). Also, there is an ability to add or not add certain source files into the list (for example, platform-specific files, or files related to features got been disabled or enabled by options)
- Also, the dirty way I did at the `tests-dev` module: I had to read all lines from the given text file and added them as source code files, and that works for both Autotools and CMake.

Reply:
OK, good to know.